lib/preview/nrk-logo-preview-nrk-buskerud-title-below.svg

```diff
@@ -0,0 +1 @@
c0-.44.085-.82.254-1.142.17-.322.41-.587.72-.796.31-.209.674-.367 1.091-.474.418-.107.88-.16 1.388-.16.204 0 .41.008.618.025.21.017.42.047.635.093.215.045.426.101.635.169.209.068.42.147.635.237l-.119 1.371a4.066 4.066 0 0 0-.49-.186c-.181-.056-.359-.107-.534-.152a5.77 5.77 0 0 0-1.007-.17 5.318 5.318 0 0 0-.406-.017c-.283 0-.534.023-.754.068-.22.045-.409.113-.567.203a.962.962 0 0 0-.355.33.878.878 0 0 0-.119.466c0 .158.037.299.11.423.074.124.17.231.288.322.119.09.251.175.398.254a3.7 3.7 0 0 0 .448.186c.164.056.333.11.508.16.175.051.35.108.525.17.175.062.353.121.533.178.248.09.485.2.711.33.226.13.423.285.593.465.169.18.302.392.398.635Zm5.536-1.368 4.105 4.627h-2.234l-3.46-4.148h-.028v4.148h-1.726V22.494l1.726-.254v7.33h.624l2.897-3.183h2.116l-4.02 4.007Zm11.587-2.915a3.068 3.068 0 0 0-1.244-.956c-.502-.215-1.075-.322-1.718-.322s-1.219.113-1.727.339c-.508.225-.94.541-1.295.948a4.15 4.15 0 0 0-.812 1.464 6.11 6.11 0 0 0-.28 1.904c0 .926.167 1.713.5 2.362a3.31 3.31 0 0 0 1.48 1.473c.655.332 1.457.499 2.405.499.451 0 .849-.025 1.193-.076.344-.051.669-.127.974-.229a6.35 6.35 0 0 0 .914-.39l.118-1.302c-.214.079-.431.157-.652.236-.22.08-.448.15-.685.212-.237.062-.48.107-.728.136a6.594 6.594 0 0 1-.745.042c-.508 0-.948-.048-1.32-.144a2.172 2.172 0 0 1-.94-.491c-.254-.231-.446-.544-.575-.94a4.28 4.28 0 0 1-.184-1.117h6.117c.079-.79.048-1.49-.093-2.099-.141-.61-.375-1.126-.703-1.549Zm-4.266.407c.362-.232.802-.348 1.32-.348.52 0 .977.127 1.304.381.328.254.556.607.686 1.058.085.291.121.61.124.948h-4.452c.042-.378.11-.713.214-.99.175-.468.443-.818.804-1.05Zm11.748-1.651-.203 1.591a3.713 3.713 0 0 0-1.185.127c-.18.051-.358.11-.533.178a4.383 4.383 0 0 0-.922.494v6.396H46.4v-8.549l1.727-.254v1.261c.04-.042.079-.087.118-.127.248-.248.514-.46.796-.634.282-.175.58-.305.897-.39.316-.084.66-.116 1.032-.093Zm7.28.237 1.726-.254-.017 8.803h-1.337l-.251-1.044c-.05.042-.099.09-.15.13-.276.225-.564.42-.863.584-.299.163-.612.287-.94.372a4.176 4.176 0 0 1-1.049.127c-.632 0-1.17-.118-1.617-.355-.445-.238-.778-.604-.998-1.1-.22-.497-.33-1.124-.33-1.88v-5.383l1.726-.254v5.332c0 .452.065.833.195 1.143.13.31.33.542.6.694.272.152.616.229 1.033.229.226 0 .443-.02.652-.06.209-.04.415-.095.618-.169.203-.073.41-.16.618-.262.127-.062.254-.133.38-.212v-6.441h.004Zm10.292-3.978v4.12a.403.403 0 0 1-.05-.024 5.828 5.828 0 0 0-2.15-.39c-.757.001-1.46.187-2.108.56-.65.372-1.171.911-1.566 1.616-.395.706-.593 1.566-.593 2.582 0 .869.15 1.62.449 2.251a3.36 3.36 0 0 0 1.286 1.465c.56.344 1.228.516 2.007.516.45 0 .877-.056 1.278-.17.4-.112.792-.287 1.176-.524.127-.08.254-.166.38-.257l.213.782h1.405V22.257l-1.727.237Zm-.906 11.08c-.445.163-.849.245-1.21.245-.542 0-1.005-.11-1.388-.33-.384-.22-.677-.559-.88-1.016-.204-.457-.305-1.03-.305-1.718 0-.689.113-1.284.338-1.752.226-.469.55-.824.974-1.067.423-.242.928-.364 1.515-.364a5.129 5.129 0 0 1 1.6.245c.081.026.174.063.262.094v5.217c-.319.189-.62.341-.906.445Z"/></g><defs><clipPath id="a"><path fill="#fff" d="M0 22h71v14H0z"/></clipPath></defs></svg>
```

Comment:
alle andre `fill=#`-regler må fjernes for at `fill="currentColor"` på rotnoden kan ta effekt.